### DISCUSSION TITLE
I can't import my Samples

### DISCUSSION THREAD

**Discussion by maxtribe** [bangcorrupt/hacktribe#108] [Q&A] [answered]
Hello folks

@bangcorrupt 

I was about to load new samples onto the Electribe. I always do this with the Electribe 2 Sample Manager, which has worked great up until now. Loaded my samples onto the file and ten seconds later I was able to program again. But my problem is that the samples on Star don't load anymore. I've tried a few methods. Anyone know what could be causing this? Because normally the samples are loaded at the start. The Electribe starts without charging briefly. I turn them on and they there at the same moment. With no sign that she's loading the samples

Many thanks in advance

Warm greetings

**Comment by bangcorrupt** [comment, marked as answer]
If the file is at `KORG/hacktribe/Sample/e2sSample.all` and the first sample is in slot 501 it should work.  If the filename or path is wrong it won't be found on boot.  If samples are in slots below 501 you will get an error.  

Does it work if you import the same file using `Data Utility > Import sample`?

**Reply by maxtribe** [reply]
@bangcorrupt The first file would be in slot 501. And for "Import All Samples" it only shows me "Unknown Error". It was still working perfectly a few days ago

**Reply by bangcorrupt** [reply]
Can you share a file demonstrating the issue?  It doesn't need lots of samples, just one to produce the error.

**Reply by maxtribe** [reply]
@bangcorrupt What kind of file do you mean

**Reply by bangcorrupt** [reply]
e2sSample.all

**Reply by maxtribe** [reply]
[e2sSample.zip](https://github.com/bangcorrupt/hacktribe/files/7920873/e2sSample.zip)

@bangcorrupt Since I have the file. fresh from the SD card

**Reply by bangcorrupt** [reply]
The first sample is in slot 19.  You need to edit the e2sSample.all so the first sample is in slot 501.

**Reply by Maxklui** [reply]
@bangcorrupt I have Edit my e2sSample.all file with the Electribe 2 Sample Manager and there was the first sample on slot 501. And what amazes me is that the Electribe doesn't show any sign of charging anything

**Reply by bangcorrupt** [reply]
I haven't used that editor. In the file posted above, the first sample is in slot 19.

[How to edit e2sSample.all](https://github.com/bangcorrupt/hacktribe/discussions/73#discussioncomment-1961200)

**Reply by ajsoriar** [reply]
In my device: "**Data Utility > Import sample**" instead of "Data Utility > Import all samples" to locate the file e2sSample.all
